cmd/call-endpoint/main.go

```diff
@@ -18,6 +18,7 @@ var args struct {
 
 	Endpoint   string   `help:"Endpoint of the desired API." arg:""`
 	Parameters []string `help:"Parameter for the endpoint in the format 'key:value', e.g., 'id:123'." arg:"" optional:""`
+	Table      bool     `help:"Attempt to output data as a TSV; defaults to JSON if there are issues." default:"false"`
```

Comment:
Nope.
The word "default" is incorrect here.
"Default" in programming means an initial value used when no value is specified.
You are using "default" all "fallback".